# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../rpm/gcc34.patch
# Copyright (C) 2004 - 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- rpm-4.1/lib/rpminstall.c.vanilla	2004-06-28 09:43:21.867093080 +0200
+++ rpm-4.1/lib/rpminstall.c	2004-06-28 09:43:46.076412704 +0200
@@ -724,7 +724,7 @@
 /*@=bounds@*/
 
 int rpmErase(rpmts ts,
-		const struct rpmInstallArguments_s * ia,
+		struct rpmInstallArguments_s * ia,
 		const char ** argv)
 {
     int count;
--- rpm-4.1/lib/rpmcli.h.vanilla	2004-06-28 09:44:37.724560992 +0200
+++ rpm-4.1/lib/rpmcli.h	2004-06-28 09:44:56.856652472 +0200
@@ -557,7 +557,7 @@
  * @param argv		array of package file names (NULL terminated)
  * @return		0 on success
  */
-int rpmErase(rpmts ts, const struct rpmInstallArguments_s * ia,
+int rpmErase(rpmts ts, struct rpmInstallArguments_s * ia,
 		/*@null@*/ const char ** argv)
 	/*@globals rpmGlobalMacroContext, fileSystem, internalState @*/
 	/*@modifies ts, ia, rpmGlobalMacroContext,
